### PR TITLE
Packaging mathcomp-analysis 0.3.0

### PR DESCRIPTION
**Pull request by affeldt-aist**
Packaging mathcomp-analysis 0.3.0

### CODE REVIEW

File: extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+build: [
+  [make "INSTMODE=global" "config"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "coq" { ((>= "8.7" & < "8.12~") | = "dev") }
+  "coq-mathcomp-field"       {(>= "1.11.0" & < "1.12~")}
```
Comment (palmskog) [inline extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.3.0/opam:23]: but `coq-mathcomp-field.1.11.0` is not in the `released` repo yet? How is this supposed to work?

Comment (palmskog) [reply extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.3.0/opam:23]: Ah, I see, it's only supposed to be beta-compatible? But is the plan then to move this package to the `released` repo, or how will clashes be avoided?

File: extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+the Coq proof-assistant and using the Mathematical Components library.
+
+It is inspired by the Coquelicot library.
+"""
+tags: [
+  "category:Mathematics/Real Calculus and Topology"
+  "keyword: analysis"
+  "keyword: topology"
+  "keyword: real numbers"
+  "logpath: mathcomp-analysis"
```
[the remark this answers is not in the document]
Comment (affeldt-aist) [reply extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.3.0/opam]: fixed

File: extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.3.0/opam (hunk 1 above)
Comment (affeldt-aist) [reply extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.3.0/opam:23]: Yet, it is only supposed to be beta-compatible.
The plan is to have the next version in `released`.

**Pull request merged**